### PR TITLE
Simplify account in deletion requests

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

- [ ] I added/updated tests.
- [x] I ensured that the PR title is good enough for the changelog.
- [x] I labeled the PR.
- [x] I self-reviewed the PR.

### CODE REVIEW

**Comment by jkoenig134** [comment]
I fear we need to do a bunch of manual testing here.

Additionally this does NOT handle the logout when an identity is deleted, we have to do that afterwards.

File: apps/enmeshed/lib/core/utils/account_utils.dart [hunk 1]
```diff
@@ -7,41 +7,14 @@ import 'package:logger/logger.dart';
...
-  return true;
-}
-
-Future<String> getAccountDeletionDate(LocalAccountDTO account) async {
-  final session = GetIt.I.get<EnmeshedRuntime>().getSession(account.id);
-  final activeIdentityDeletionResult = await session.transportServices.identityDeletionProcesses.getActiveIdentityDeletionProcess();
-  return activeIdentityDeletionResult.value.gracePeriodEndsAt!;
-}
-
 Future<List<LocalAccountDTO>> getAccountsInDeletion() async {
```
Comment (Milena-Czierlinski) [inline apps/enmeshed/lib/core/utils/account_utils.dart:10]: I also added functions for this and `getAccountsNotInDeletion` to the MultiAccountController.

Comment (jkoenig134) [reply apps/enmeshed/lib/core/utils/account_utils.dart:10]: Nice! I'll add this!

File: apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_and_choose_next.dart [hunk 2]
```diff
@@ -71,7 +71,7 @@ class _DeleteProfileAndChooseNextState extends State<DeleteProfileAndChooseNext>
       );
 
   Future<void> _loadAccounts() async {
-    final accounts = await getAccountsNotInDeletion();
+    final accounts = await GetIt.I.get<EnmeshedRuntime>().accountServices.getAccountsNotInDeletion();
```
Comment (aince42) [inline apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_and_choose_next.dart:74]: Why did you decide to write GetIt and the function call on separate lines in some places and sometimes on one line?

Comment (jkoenig134) [reply apps/enmeshed/lib/profiles/profile/modals/delete_profile/delete_profile_and_choose_next.dart:74]: There are two reasons to split the calls:

1. I use the runtime more than once
2. the line gets too long

**Review by aince42 — APPROVED** [review]

**Review by Milena-Czierlinski — APPROVED** [review]

**Pull request merged**